app/segments/drill/[id]/submission/input.js

```diff
@@ -282,14 +357,34 @@ function createOutputData(
   return outputData;
 }
 
+//A function to validate inputs are all numbers
+function validateInputs(inputs) {
+  const inputKeys = Object.keys(inputs);
```

Comment:
return Object.values(inputs).some((input)=>isNaN(input))

Reply:
Tried this and it allowed this to go through...
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/02c88794-852b-4ce9-932b-c6a2984bd1c7)
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/750171d0-dbb6-4fea-8a59-fc33fb0d7d3f)

Reply:
had to adjust it in the handling function as well to account for the return type. should be working now